### ISSUE TITLE
Registry Finder 2.57 - Installer and language file issue

### ISSUE THREAD

**Issue by bovirus** [registry-finder/translations#3]
@rstain 

I downloaded Registry Finder 2.57 and I found some issues

**Installer**

Installer, based on Inno Setup, Probably not use 

UsePreviousLanguage=no

in [Setup] section.
Then installing new version, get previous language settings (where italian language was not available) and set as default english language for the installer instead Italian.

Please set

UsePreviousLanguage=no

![Registry Finder - Installer exe properties](https://github.com/registry-finder/translations/assets/1262554/27c450e1-b3a2-4246-b38d-4f494d4750ba)

In installer exe file properties Copyright string is too lomng and is truncated.
Please remove "Copyright (C)" (copyright is 

**Language files**

There are some mismatches about language files available in website 

https://registry-finder.com/translations.html

and language files available on github

https://github.com/registry-finder/translations

The file in website are  ending with CR/LF and file on github ending with LF only.

In website is availble Russian file 2.57 that is not available in github.

In github is availble english file (master) che is not availble in website.

**Comment by registry-finder** [comment]
Indeed, I do not use UsePreviousLanguage.
Well, you have to change (the default) English to Italian only once, right? Next time installer will use Italian by default.
As I understand the documentation, UsePreviousLanguage=no will cause installer to use English as default every time. I guess, that is not what you want. Did I miss something?

Truncation of copyright is not a big deal. I examined several dlls including MS - vast majority of them have the "Copyright" word and are truncated. It seems, nobody cares. On the other hand, to remove "copyright" is not a big deal, as well. We'll see...

I will examine language files later.

**Comment by bovirus** [comment]
Hi.

No. UsePreviousLanagiueg=no doesn'tmean that use english by default.

The scope is different.
Don't consider settings of previous instaltaion (e.x select english why not available italian that is available in new version).

The scope of UsePreviousLanagueg=no is to re-enable language detection matching bewteen OS language and installer language available.

That it's important option.

If Microsft make an error (truncating etxt) are you sure that you want  to follow the error?
Are you sure that soemone reported the error to  MS?
Someone before me report this isssue to you about your program?

I believe that is more important in Copyright to see your name then the Word "(c) Copyright"
For me the non sense is that "(c)" and "Copyright" are synonyms and the field name is "Copyright".
It's that "Copyright" is field name but, for me, "(c)" and "Copytright" as starting copyright messagein this context (and truncating author/company name)  is a non sense.

Thanks. KR.

________________________________
Da: registry-finder ***@***.***>
Inviato: mercoledì 24 maggio 2023 23:21
A: registry-finder/translations ***@***.***>
Cc: Roberto Boriotti ***@***.***>; Author ***@***.***>
Oggetto: [FROM EXTERNAL] Re: [registry-finder/translations] Registry Finder 2.57 - Installer and language file issue (Issue #3)


Indeed, I do not use UsePreviousLanguage.
Well, you have to change (the default) English to Italian only once, right? Next time installer will use Italian by default.
As I understand the documentation, UsePreviousLanguage=no will cause installer to use English as default every time. I guess, that is not what you want. Did I miss something?

Truncation of copyright is not a big deal. I examined several dlls including MS - vast majority of them have the "Copyright" word and are truncated. It seems, nobody cares. On the other hand, to remove "copyright" is not a big deal, as well. We'll see...

I will examine language files later.

**Comment by registry-finder** [comment]
@bovirus 

>The scope of UsePreviousLanagueg=no is to re-enable language detection matching bewteen OS language and installer language available.

I do not get the idea, please clarify.
For better understanding, lets consider concrete scenarios. I can see two base use cases.
1. The OS language is, say, Russian. I want Registry Finder to be in English. (I guess, that is quite common use case. Personally, I almost always prefer non localized versions vs localized.)
Assume I was installed 2.57 English and now installing 2.58.
Then, if UsePreviousLanguage=no, the default setup language will be Russian - not what I want. If UsePreviousLanguage=yes then the default will be English - OK. 
2. The OS language is Russian and I prefer want Registry Finder to be in Russian.
I was installed 2.57 Russian and now installing 2.58.
Regardless of the value of UsePreviousLanguage, the default setup language will be Russian - OK.
So what the point of setting UsePreviousLanguage=no?

> The file in website are ending with CR/LF and file on github ending with LF only.

I do not see any bad consequences, but I changed line endings to CRLF just in case.

> In website is availble Russian file 2.57 that is not available in github.

Updated Russian and English to 2.57.

> In github is availble english file (master) che is not availble in website.

Would it be helpful for anyone? I doubt it.

> It's that "Copyright" is field name but, for me, "(c)" and "Copytright" as starting copyright messagein this context (and truncating author/company name)  is a non sense.

Okay, in that context the "Copyright" and "(c)" are redundant. I will remove the first one. "(c)" will still be redundant, but it is required in the context of the About dialog.

**Comment by bovirus** [comment]
@registry-finder 

<img width="598" height="459" alt="Image" src="https://github.com/user-attachments/assets/24b3fcb0-2d3a-49cf-899b-2028d7839e76" />

In Regsitry Finder Installer the string "versione" on top left is not necessary.

To fix this issue (already reported to Inno Setup author and fixed in Inno Setup 7.x), please add in custom message for Italian strings

[CustomMessages]
xxxx.NameAndVersion=%1 %2

where xxxx is Italian custom message language id

**Comment by registry-finder** [comment]
Thank you for the suggestion. I will try it tomorrow.

**Comment by registry-finder** [comment]
Yes, I tried it, and it works. But the issue is not serious at all, so I'm not keen on rebuilding and republishing the installer. This fix will be included in the next version.

**Comment by bovirus** [comment]
@registry-finder 

Sure. You can apply at next build.
Note: you can apply the same fix for English and other lnguages.
